$NetBSD: patch-Makefile.orig,v 1.1 2021/02/21 12:33:32 martin Exp $

realpath is not very portable, do not override install dirs

--- Makefile.orig	2020-05-17 05:45:34.000000000 +0200
+++ Makefile	2021-02-20 19:20:33.863583131 +0100
@@ -1,10 +1,7 @@
 all: imx_usb imx_uart
 
-DESTDIR ?=
-prefix ?= /usr
-bindir ?= $(prefix)/bin
-sysconfdir ?= $(prefix)/etc
-rel_sysconfdir = $(shell realpath -m --relative-to=$(bindir) $(sysconfdir))
+bindir=$(PREFIX)/bin
+sysconfdir=$(PREFIX)/etc
 
 BUILDHOST := $(shell uname -s)
 BUILDHOST := $(patsubst CYGWIN_%,CYGWIN,$(BUILDHOST))
@@ -17,7 +14,7 @@
 USBCFLAGS = -I/usr/include/libusb-1.0
 USBLDFLAGS = -L/usr/lib -lusb-1.0
 endif
-CONFCPPFLAGS = -DSYSCONFDIR='"$(sysconfdir)"' -DREL_SYSCONFDIR='"$(rel_sysconfdir)"'
+CONFCPPFLAGS = -DSYSCONFDIR='"$(sysconfdir)"' -DREL_SYSCONFDIR='"$(sysconfdir)"'
 CFLAGS ?= -Wall -Wstrict-prototypes -Wno-trigraphs
 
 imx_usb.o : imx_usb.c imx_sdp.h imx_loader_config.h portable.h
@@ -33,16 +30,16 @@
 	$(CC) -o $@ $^ $(LDFLAGS)
 
 install: imx_usb imx_uart
-	mkdir -p '$(DESTDIR)$(sysconfdir)/imx-loader.d/'
-	install -m644 *.conf '$(DESTDIR)$(sysconfdir)/imx-loader.d/'
-	mkdir -p '$(DESTDIR)$(bindir)'
-	install -m755 imx_usb '$(DESTDIR)$(bindir)/imx_usb'
-	install -m755 imx_uart '$(DESTDIR)$(bindir)/imx_uart'
+	mkdir -p '$(sysconfdir)/imx-loader.d/'
+	install -m644 *.conf '$(sysconfdir)/imx-loader.d/'
+	mkdir -p '$(bindir)'
+	install -m755 imx_usb '$(bindir)/imx_usb'
+	install -m755 imx_uart '$(bindir)/imx_uart'
 
 uninstall:
-	rm -rf '$(DESTDIR)$(sysconfdir)/imx-loader.d/'
-	rm -rf '$(DESTDIR)$(bindir)/imx_usb'
-	rm -rf '$(DESTDIR)$(bindir)/imx_uart'
+	rm -rf '$(sysconfdir)/imx-loader.d/'
+	rm -rf '$(bindir)/imx_usb'
+	rm -rf '$(bindir)/imx_uart'
 
 clean:
 	rm -f imx_usb imx_uart *.o
